recipes/package_repo.rb

```diff
@@ -97,7 +97,11 @@
   when 'debian'
     apt_repository "mongodb-org-#{pkg_major_version}" do
       uri node['mongodb3']['package']['repo']['url']
-      distribution "#{node['lsb']['codename']}/mongodb-org/#{node['mongodb3']['package']['repo']['apt']['name']}"
+      if node['platform'] == 'ubuntu' and node['platform_version'].to_f >= 15.04
+        distribution "trusty/mongodb-org/#{node['mongodb3']['package']['repo']['apt']['name']}"
```

Comment:
Right, seems so they support only LTS versions:
[Platform Support](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/)
> MongoDB only provides packages for 64-bit long-term support Ubuntu releases. Currently, this means 12.04 LTS (Precise Pangolin) and 14.04 LTS (Trusty Tahr). While the packages may work with other Ubuntu releases, this is not a supported configuration.

I agree that support of 15.04 might be considered as a hack, but mongodb release for 14.04 works on 15.04 as well. I will fix this condition and force trusty/mongodb-org release only for 15.04